src/metpy/calc/thermo.py

```diff
@@ -2574,3 +2574,55 @@ def specific_humidity_from_dewpoint(pressure, dewpoint):
     """
     mixing_ratio = saturation_mixing_ratio(pressure, dewpoint)
     return specific_humidity_from_mixing_ratio(mixing_ratio)
+
+
+@exporter.export
+@preprocess_xarray
+@check_units('[pressure]', '[temperature]', '[temperature]')
+def lifted_index(pressure, temperature, parcel_profile):
+    """Calculate Lifted Index from the pressure temperature and parcel profile.
+
+    Lifted index formula derived from [Galway1956] and referenced by [Doswell-Schultz2006]:
```

Comment:
So if you add a trailing underscore, as in `[Galway1956]_`, it can link to our existing page of references.

Reply:
I can do that as well. Is the page of references viewable? How about Doswell and Schultz? So I don't need to copy the links. Just put the underscore after the references.